lib/users/routes.js

```diff
@@ -261,9 +312,46 @@ const save = (req, res, next) => {
   return res.redirect(referrer);
 };
 
+/**
+ * Save first user
+ * @param {object} req - HTTP Request
+ * @param {object} res - HTTP Response
+ * @param {object} next - Express callback
+ *
+ * @returns {promise} database promise
+ *
+ */
+const first = (req, res, next) => {
+  return database.select('*').from('users').then(rows => {
+    // if any users in DB, we give them a shameful error
+    if (rows.length > 1) {
```

Comment:
I think I'm confused? Or perhaps you are?

`first` is for saving the first user (or superadmin as we may do). It's really just a wrapper for `save` with this db check before triggering `save`. Here in case someone tries to `post` to this url (`/create-admin`) when they shouldn't. 

i.e. this db check _is_ the access control you want, isn't it?